.github/workflows/ci-rust.yml

```diff
@@ -266,6 +291,7 @@ jobs:
           prerelease: false
           files: |
             artifacts/selenium-manager-linux/selenium-manager-linux
+            artifacts/selenium-manager-linux-arm64/selenium-manager-linux-arm64
```

Comment:
Realizing I forgot to add `artifacts/selenium-manager-linux-arm64-debug/selenium-manager-linux-arm64-debug.tar` to this list, so let's keep that in mind as we continue working on this PR.